tools/perf/ucc_pt_config.cc

```diff
@@ -0,0 +1,86 @@
+#include "ucc_pt_config.h"
+
+ucc_pt_config::ucc_pt_config() {
+    bootstrap.bootstrap = UCC_PT_BOOTSTRAP_MPI;
+    bench.coll_type     = UCC_COLL_TYPE_ALLREDUCE;
+    bench.min_count     = 128;
+    bench.max_count     = 128;
+    bench.dt            = UCC_DT_FLOAT32;
+    bench.mt            = UCC_MEMORY_TYPE_HOST;
+    bench.op            = UCC_OP_SUM;
+    bench.inplace       = false;
+    bench.n_iter        = 10;
```

Comment:
I think it is very useful to have iter/warmup pair for at least "small" and "large" msg sizes. For small (say up to 64K) we want warmup 100, iter 1000 i beleive. 10/10 will have a lot of noise on high core counts